### PR TITLE
Add hero block support and refactoring

### LINKED ISSUE #1: Implement renderHero() function in copilot.js
`renderHero()` at https://github.com/sdmcraft/helix-project-boilerplate/blob/emea-summit/blocks/copilot/copilot.js#L13 needs to be implemented

It accepts `block_name`, `content` as arguments where content is a json object as following:

> {
> text : some AI generated text,
> image : some AI generated Image URL
> }

### PR DESCRIPTION
**Pull request by sdmcraft**
Fix #1 

Test URLs:
- Before: https://emea-summit-updated--helix-project-boilerplate--sdmcraft.hlx.page/copilot
- After: https://emea-summit--helix-project-boilerplate--sdmcraft.hlx.page/copilot

### CODE REVIEW

File: blocks/copilot/copilot.js [hunk 1]
```diff
@@ -105,76 +97,81 @@ export default function decorate(block) {
...
+		// Get the input value
+		const promptStr = document.getElementById('prompt').value;
+		//Prepare chat api
+		console.log("[copilot] Initiate chat with Azure to train it to parse prompt to JSON");
+		const conversation = await initiateChat();
+		console.log("[copilot] Chat initiated");
+		//Call chat api
+		conversation.push({
+			role: 'user',
+			content: `Parse and return only the JSON representation for "Create ${promptStr}" without anything else`,
```
Comment (vnagpal81) [inline blocks/copilot/copilot.js:143]: @sdmcraft Does prefixing Create before prompt work well, if prompt is "Build a website...."?

File: blocks/copilot/ims.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-function getAccessToken(cfg = {}) {
-	if(!cfg.token) {
-		//call IMS API
-	}
-	return cfg.token;
+export default function getAccessToken(cfg = {}) {
+  if (!cfg.token) {
```
Comment (vnagpal81) [inline blocks/copilot/ims.js:2]: should be add the ims api code also if some one wants to use the instance later?

**Review by vnagpal81 — APPROVED** [review]
lgtm

File: blocks/copilot/copilot.js (hunk 1 above)
Comment (sdmcraft) [reply blocks/copilot/copilot.js:143]: yes probably "Create..." is not needed as we don't use the `action` for now anyways. I'll remove it.

File: blocks/copilot/ims.js (hunk 2 above)
Comment (sdmcraft) [reply blocks/copilot/ims.js:2]: Ideally yes, though probably for demo we can just use the hardcoded token.

**Pull request merged**